django_mongodb/base.py

```diff
@@ -170,7 +172,10 @@ def get_connection_params(self):
         }
 
     def get_new_connection(self, conn_params):
-        return MongoClient(**conn_params)
+        return MongoClient(**conn_params, driver=self._driver_info())
+
+    def _driver_info(self):
+        return DriverInfo("django-mongodb", metadata.version("django-mongodb"))
```

Comment:
Rather than using `importlib.metadata` you could use `from . import __version__ as django_mongodb_version` (this is what I used in other libraries).